Makefile.am

```diff
@@ -38,7 +38,6 @@ EXTRA_DIST = \
 	INSTALL \
 	QUICKSTART \
 	README \
-	SPONSORS \
```

Comment:
Moving SPONSORS away from the similar CONTRIBUTORS file feels wrong to me. Do we have to move SPONSORS.in to `doc/`? Does keeping SPONSORS.in in root source directory break something?

We could move both files to `doc/`, of course, but I think these people and companies deserve a top-level mention.